models/node_model.go

```diff
@@ -31,10 +31,11 @@ type NodeType string
 
 // Node types (in the same order as in nodes.proto).
 const (
-	GenericNodeType   NodeType = "generic"
-	ContainerNodeType NodeType = "container"
-	RemoteNodeType    NodeType = "remote"
-	RemoteRDSNodeType NodeType = "remote_rds"
+	GenericNodeType             NodeType = "generic"
+	ContainerNodeType           NodeType = "container"
+	RemoteNodeType              NodeType = "remote"
+	RemoteRDSNodeType           NodeType = "remote_rds"
+	RemoteAzureDatabaseNodeType NodeType = "remote_azure_database"
```

Comment:
Just to keep the same logic, RDS is also a database but it doesn't have the "database" word at the end.
I would set the node type to just `remote_azure`

Reply:
@percona-csalguero We discuss it with @askomorokhov and agree, but in scope of Azure you can also have another services than database.